sycl/test/basic_tests/sycl_maximum_minimum.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clangxx -fsycl -fsyntax-only %s
```

Comment:
Currently this test checks nothing. It has asserts in it, but this compilation line does not produce a binary, and binary execution line is missing.

Reply:
Let me fix this too. Thanks!

Reply:
Why it's marked as resolved? There is no fix in this PR.